packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -3019,6 +3028,17 @@ export const audiusBackend = ({
     return waudioBalance
   }
 
+  /**
+   * Fetches the Sol balance for the given wallet address
+   * @param {string} The solana wallet address
+   * @returns {Promise<BNWei>}
+   */
+  async function getAddressSolBalance(address: string): Promise<BNWei> {
```

Comment:
@piazzatron / @jowlee I wasn't aware of this `AudiusBackend.ts` / `WalletClient.ts` pattern before - should we continue it? Feels like `WalletClient.ts` could just call on libs itself?

Reply:
Also should we be using `WalletAddress` types here?